lib/functions/modem.js

```diff
@@ -191,7 +199,67 @@ module.exports = function (SerialPort) {
   }
 
   modem.initializeModem = function (callback, priority, timeout) {
-    if (callback == undefined) {
+
+    function sendAdditionalCommands() {
```

Comment:
neat!!

Reply:
yeeay, else it got too complex for the two cases with and without PIN